lib/screens/sign_up_screen.dart

```diff
@@ -21,23 +21,146 @@ class _SignUpScreenState extends State<SignUpScreen> {
         backgroundColor: const Color(0xFFF3ACB5),
       ), 
       backgroundColor: Color(0xFF500F15),
-      body: const Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: 
-          [
-            // const SizedBox(height: 80),
-            Text("Sign Up",
-              style: TextStyle(
-                color: Color(0xFFFCD7DD),
-                fontSize: 20,
-                fontFamily: "Fredoka",
-                fontWeight: FontWeight.bold,
-              ),
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Text(
+            'Sign Up',
+            style: TextStyle(
+              fontSize: 35,
+              color:  Color(0xFFF3ACB5),
+              fontFamily: "Fredoka",
+              fontWeight: FontWeight.bold,
+
             ),
+          ),
+
+          Padding(
+            padding: const EdgeInsets.symmetric(vertical: 25.0),
+            child: Form(
+              child: Column(
+                children: [
+            
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 20.0),
+                    child: TextFormField(
+                      style: TextStyle(color: Colors.white),
+                      keyboardType: TextInputType.emailAddress,
+                      decoration: InputDecoration(
+                        labelText: 'Username',
+                        labelStyle: TextStyle(color: Colors.white),
```

Comment:
need to make sure we have the same font for the text here as well as the font. For login i have fontSize: 15,
fontFamily: "Montserrat-Regular",